demos/common/export_models/export_model.py

```diff
@@ -0,0 +1,361 @@
+output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+
+node: {
+  name: "LLMExecutor"
+  calculator: "HttpLLMCalculator"
+  input_stream: "LOOPBACK:loopback"
+  input_stream: "HTTP_REQUEST_PAYLOAD:input"
+  input_side_packet: "LLM_NODE_RESOURCES:llm"
+  output_stream: "LOOPBACK:loopback"
+  output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+  input_stream_info: {
+    tag_index: 'LOOPBACK:0',
+    back_edge: true
+  }
+  node_options: {
+      [type.googleapis.com / mediapipe.LLMCalculatorOptions]: {
+          models_path: "{{model_path}}",
+          plugin_config: '{"KV_CACHE_PRECISION": "{{kv_cache_precision|default("", false)}}"}',
+          block_size: {{block_size|default("32", true)}},
+          enable_prefix_caching: {{enable_prefix_caching|default("true", true)}},
+          cache_size: {{cache_size|default("10", true)}},
+          max_num_batched_tokens: {{max_num_batched_tokens|default("8120", false)}},
+          max_num_seqs: {{max_num_seqs|default("256", true)}},
+          device: "{{device|default("CPU", true)}}",
+
+      }
+  }
+  input_stream_handler {
+    input_stream_handler: "SyncSetInputStreamHandler",
+    options {
+      [mediapipe.SyncSetInputStreamHandlerOptions.ext] {
+        sync_set {
+          tag_index: "LOOPBACK:0"
+        }
+      }
+    }
+  }
+}"""
+
+embeddings_subconfig_template = """{
+    "model_config_list": [
+    { "config": 
+	    {
+                "name": "{{model_name}}_tokenizer_model",
+                "base_path": "tokenizer"
+            }
+	},
+    { "config": 
+	    {
+                "name": "{{model_name}}_embeddings_model",
+                "base_path": "embeddings",
+                "target_device": "{{target_device|default("CPU", true)}}",
+                "plugin_config": { "NUM_STREAMS": {{num_streams|default("1", true)}} }
+            }
+	}
+   ]
+}"""
+
+rerank_subconfig_template = """{
+    "model_config_list": [
+    { "config": 
+	    {
+                "name": "{{model_name}}_tokenizer_model",
+                "base_path": "tokenizer"
+            }
+	},
+    { "config": 
+	    {
+                "name": "{{model_name}}_rerank_model",
+                "base_path": "rerank",
+                "target_device": "{{target_device|default("CPU", true)}}",
+                "plugin_config": { "NUM_STREAMS": {{num_streams|default("1", true)}} }
+            }
+	}
+   ]
+}"""
+
+
+def set_rt_info(model_folder_path, model_filename, config_filename):
+    model = ov.Core().read_model(os.path.join(model_folder_path, model_filename))
+    with open(os.path.join(model_folder_path, config_filename), 'r') as config_file:
+        config_data = json.load(config_file)
+        for key, value in config_data.items():
+          try:
+              model.set_rt_info(value, ['config', key])
+          except Exception as e:
+              model.set_rt_info(str(value), ['config', key])
+    temp_model_name = model_filename.replace('.xml', '_temp.xml')
```

Comment:
Why do we need to save model with different name and then replace original one? Isn't it possible to simply overwrite original model in 1 go using ov.save_model?

Reply:
it is not possible in ov api - exception is raised